<launch>
 <group ns="lidar1">
  <node name="lidar01" pkg="bluesea2" type="bluesea2_node" output="screen" >
    #ROS#
    <param name="scan_topic" value="scan"/>
    <param name="cloud_topic" value="cloud"/>
    <param name="frame_id" value="map" />
     #DATA#
    <param name="min_dist" value="0.1"/>
    <param name="max_dist" value="80.0"/>
    <param name="output_scan" value="true" />
    <param name="output_cloud" value="false"/>
    <param name="output_cloud2" value="false"/>
    <param name="reversed" value="false"/>
    <param name="hard_resample" value="false"/>
    <param name="soft_resample" value="false"/>
    <param name="with_angle_filter" value="false"/>
    <param name="min_angle" value="-1"/>
    <param name="max_angle" value="2"/> 
    <param name="with_confidence" value="1"/>
    <param name="unit_is_mm" value="0"/>
    <!-- <rosparam param="mask1" >[0,3.14]</rosparam> -->
    <!-- <rosparam param="mask2" >[-1,0]</rosparam> -->
    #CONNECT#
    <param name="type" value="udp" />
    <param name="lidar_port" value="5000" />
    <param name="local_port" value="6668" />
    <param name="lidar_ip" value="192.168.158.91"/>
    #SET#
    <param name="rpm" value="600"/>
  </node>
 </group>
</launch>
